app/packages/core/src/plugins/SchemaIO/components/TableView.tsx

```diff
@@ -47,23 +71,70 @@ export default function TableView(props) {
                     {label}
                   </TableCell>
                 ))}
+                {hasRowActions && <TableCell />}
               </TableRow>
             </TableHead>
             <TableBody {...getComponentProps(props, "tableBody")}>
-              {table.map((item) => (
+              {rows.map((item, rowIndex) => (
                 <TableRow
                   key={item.id}
                   sx={{ "&:last-child td, &:last-child th": { border: 0 } }}
                   {...getComponentProps(props, "tableBodyRow")}
                 >
-                  {columns.map(({ key }) => (
-                    <TableCell
-                      key={key}
-                      {...getComponentProps(props, "tableBodyCell")}
-                    >
-                      {item[key]}
+                  {columns.map(({ key }, columnIndex) => {
+                    const coordinate = [rowIndex, columnIndex].join(",");
+                    const isSelected =
+                      selectedCells.has(coordinate) ||
```

Comment:
I have been running into some of these errors when a tableview does not have selectedRows/Columns/Cells in the initial state. 

![image](https://github.com/user-attachments/assets/f13c4739-1637-4742-9c06-521441bbeaf2)